v2/pkg/protocols/common/interactsh/interactsh.go

```diff
@@ -50,7 +53,7 @@ type Client struct {
 
 var (
 	defaultInteractionDuration = 60 * time.Second
-	interactshURLMarker        = "{{interactsh-url}}"
+	interactshURLMarkerRegex   = regexp.MustCompile(`{{interactsh-url(?:-[0-9]+)}}`)
```

Comment:
The corresponding test seems to fail with basic placeholders `{{interactsh-url}}`, not sure if this might be an option:
```go
var interactshURLMarkerRegex = regexp.MustCompile(`(?m){{interactsh-url([0-9]*)}}`)
```